.github/workflows/make_bundle_conda.yml

```diff
@@ -4,6 +4,8 @@ on:
   pull_request:
     branches:
       - main
+    paths:
+      - 'build_installers.py'
```

Comment:
Curious about the rationale behind constraining the runs to this file only. otherwise PR lgtm

Reply:
Hi @potating-potato, so we do not want to trigger a bundle build unless something changed in the file in charge of creating the bundles. I added this on this PR to make the other PRs simpler.